tests/core/pyspec/eth2spec/test/eip7594/unittests/polynomial_commitments/test_polynomial_commitments.py

```diff
@@ -48,12 +55,14 @@ def test_verify_cell_proof_batch(spec):
     commitment = spec.blob_to_kzg_commitment(blob)
     cells, proofs = spec.compute_cells_and_proofs(blob)
 
+    cells_bytes = [[field_element_bytes(element) for element in cell] for cell in cells]
+
     assert spec.verify_cell_proof_batch(
-        row_commitments=[commitment],
+        row_commitments_bytes=[commitment],
         row_ids=[0],
         column_ids=[0, 1],
-        cells=cells[0:1],
-        proofs=proofs,
+        cells_bytes=cells_bytes[0:1],
+        proofs_bytes=proofs,
```

Comment:
We're sending all the proofs when only the first one is required.

Reply:
Resolved in 2000a4f30